include/global.h

```diff
@@ -25,6 +25,14 @@
 #define INCBIN_S32 {0}
 #endif // IDE support
 
+// Invalid / Out of Bound Placeholder values
+#define INVALID_U8 0xFF
+#define INVALID_U16 0xFFFF
+#define INVALID_U32 0xFFFFFFFF
+#define INVALID_S8 -1
+#define INVALID_S16 -1
+#define INVALID_S32 -1
```

Comment:
Is there any reason not to make `INVALID_S8`, `INVALID_S16`, and `INVALID_S32` all a single `INVALID_SIGNED` define?

Reply:
These constants were already removed from this PR.